src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Star, StarOutline } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+export const StarRating = ({
+  fillColor,
+  name,
+  label,
+  scale = 5,
+  value,
+  onChange,
+  outlineColor,
+  ...rest
+}) => {
+  const [rating, setRating] = useState(value);
+
+  const options = [];
+  for (let i = 0; i < scale; i += 1) {
+    options.push(i);
+  }
+
+  return (
+    <RadioButtonGroup
+      direction="row"
+      options={options}
+      value={rating}
+      name="starRating"
+      onChange={(event) => {
+        const adjustedRating = parseInt(event.target.value, 10) + 1;
```

Comment:
what happens if someone passes their own `options`  here? will this calculation cause any issues?

Reply:
or are we discouraging people from passing their own options? which I'm fine with too.

Reply:
yeah Im thinking of changing this to either have this as default or accept `options`

Reply:
This might be what is causing the weird keyboard behavior where it is off by one

Reply:
Thinking we dont let someone pass `options` for now first pass since Im not sure what other `options` they would be passing since this is just a rating and they can change the `scale`